modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainerV1.java

```diff
@@ -0,0 +1,164 @@
+package org.testcontainers.containers;
+
+import java.util.Collections;
+import java.util.Set;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * See <a href="https://store.docker.com/images/influxdb">https://store.docker.com/images/influxdb</a>
+ */
+class InfluxDBContainerV1<SELF extends InfluxDBContainerV1<SELF>> extends GenericContainer<SELF> {
```

Comment:
Shouldn't this be public? My thinking is:

* We don't want people using `InfluxDBContainer`, but the reason for that is that the version is ambiguous. It's not because we want to drive people away from using InfluxDB v1 - they might have good reasons for it (like it's part of their production infrastructure that they need to carry on supporting).
* People might want to switch to using `InfluxDBContainerV1` to avoid the deprecation warning, but they need to carry on using InfluxDB v1 for testing.

Reply:
That is a good point! I will change this to public!